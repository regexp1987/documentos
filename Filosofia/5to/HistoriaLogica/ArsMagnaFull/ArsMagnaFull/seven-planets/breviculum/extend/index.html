<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="Author" content="Le Myésier, Raymond Lull, translated by Y. Dambergs">
   <meta name="GENERATOR" content="Mozilla/4.79 [en] (Win98; U) [Netscape]">
   <title>Extended Overview of the Whole Breviculum</title>
</head>
<body background="white.jpg">
&nbsp;
<center>
<p><font size=+2>Extended Overview of the Entire Breviculum</font></center>

<p><br>
<center><table BORDER=0 COLS=2 WIDTH="35%" >
<tr>
<td><img SRC="totduo.jpg" HSPACE=1 VSPACE=1 BORDER=1 height=194 width=179></td>

<td><img SRC="totu.gif" HSPACE=1 VSPACE=1 BORDER=1 height=194 width=177></td>
</tr>

<tr>
<td>Photocopy of chart from the 14th century Karlsruhe MS&nbsp;
<br>of Le Myesier's Breviculum</td>

<td VALIGN=TOP>The same chart reproduced with MindManager software
<br>1998&nbsp;</td>
</tr>
</table></center>

<center>
<p><font face="Arial,Helvetica"><font color="#000099"><font size=-1><a href="extmmp.htm">VIEW
THE WHOLE MINDMAP</a></font></font></font><font face="Arial,Helvetica"></font>
<p><font face="Arial,Helvetica"><font color="#000099"><font size=-1><a href="1extend1.htm">VIEW
MULTI MAP</a></font></font></font>
<br><font color="#CC0000">(navigate multimap by clicking</font>
<br><font color="#CC0000">on hyperlinks and using browser's back button)</font>
<p><b><a href="../golden/goldfig.htm">Previous</a></b>
<br><b><a href="../real/index.html">Next</a></b>
<br><b><a href="../../seven-planets.htm">Contents</a></b></center>

</body>
</html>
