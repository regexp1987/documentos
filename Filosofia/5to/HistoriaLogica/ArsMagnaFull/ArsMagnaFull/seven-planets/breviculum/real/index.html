<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="Author" content="default">
   <meta name="GENERATOR" content="Mozilla/4.79 [en] (Win98; U) [Netscape]">
   <title>Real Being in Existence and Action</title>
</head>
<body background="white.jpg">
&nbsp;
<center>
<p><font size=+3>Real Being in Existence and Appearance</font></center>

<p><br>
<center><table BORDER=0 CELLPADDING=6 COLS=2 WIDTH="17%" >
<tr>
<td><img SRC="ensms2.gif" HSPACE=1 VSPACE=1 BORDER=1 height=199 width=181></td>

<td><img SRC="ensov.gif" HSPACE=1 VSPACE=1 BORDER=1 height=199 width=184></td>
</tr>

<tr>
<td>Photocopy of chart from the14th century Karlsruhe MS of Le Myesier's
Breviculum</td>

<td>The same chart reproduced&nbsp;&nbsp;&nbsp; with MindManager software&nbsp;
<br>1998&nbsp;</td>
</tr>
</table></center>

<center><font face="Arial"><font size=-1><a href="realmp.htm">VIEW THE
WHOLE MINDMAP</a></font></font>
<p><font face="Arial"><font size=-1><a href="areal.htm">VIEW MULTI MAP</a></font></font>
<br><font face="Arial"><font color="#FF0000"><font size=-1>navigate multimap
by clicking on links</font></font></font>
<br><font face="Arial"><font color="#FF0000"><font size=-1>and using browser's
back button</font></font></font></center>

<blockquote>
<blockquote>
<blockquote>
<blockquote><font face="Arial"><font color="#000000"><font size=-1>This
second figure is really born of the first or Golden Figure which is its
mother as it were, and the first figure was pregnant with the second figure
which it implicitly contained.&nbsp; Now&nbsp; when you see an object at
a great distance, you receive a universal impression of it that does not
yet reveal what it is, but only some confused, unknown but real entity,
substance or body. Then as&nbsp; you get closer, it can be identified as
an animal by its perceptible shape and animal movement. And getting even
closer, you can see that it is a man. Similarly as the eye approaches this
figure it becomes obvious that it was implicitly contained in the first
figure where it existed in apparent confusion.</font></font></font>
<p><font face="Arial"><font color="#000000"><font size=-1>Here we see its
introduction, origin and the subsequent order it follows. Through this
second figure, the first or Golden Figure displays, moves and unfolds its
members and shows the intrinsic parts of its being through gradual connections.
And this unfoldment can be learned through the sense of sight and captured
by the imagination and kept in memory until it is again remembered and
communicated to the intellect. The necessity of this figure becomes apparent
as we look at it, for the general scope of the first figure needs to be
clarified and opened by using the second as an auxiliary. It is useful
for visualizing the divisions of being without which no knowable object
can be distinguished as without these divisions the human intellect remains
in total ignorance and confusion.</font></font></font>
<p><font face="Arial"><font color="#000000"><font size=-1>I made these
visual figures so that by viewing them you can immediately and clearly
see many things and commit them to memory all in one time and place as
you memorize the figures by imagining them. And as memory brings them back
to the intellect, the will is moved to love the figures. And thus the soul
takes great delight in learning sciences while visualizing them.</font></font></font>
<p><font face="Arial"><font color="#000000"><font size=-1>The final conclusion
of the above is that any conclusion that destroys the order of figures
or any figure or member is contrary to the truth, and indicates that the
true conclusion must be signified by its opposite. If the false conclusion
is affirmative, then take its negative version and if it is negative, take
the affirmative version of it as the true conclusion.</font></font></font></blockquote>
</blockquote>
</blockquote>
</blockquote>

<center>
<p><br><b><font face="Times New Roman,Times"><font color="#FF0000"><a href="../extend/index.html">Previous</a></font></font></b>
<br><b><font face="Times New Roman,Times"><font color="#FF0000"><a href="../homo1.htm">Next</a></font></font></b>
<br><b><font face="Times New Roman,Times"><font color="#FF0000"><a href="../../seven-planets.htm">Contents</a></font></font></b></center>

</body>
</html>
