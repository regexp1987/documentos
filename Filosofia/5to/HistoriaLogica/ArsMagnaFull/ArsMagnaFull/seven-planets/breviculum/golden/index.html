<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="GENERATOR" content="Mozilla/4.79 [en] (Win98; U) [Netscape]">
   <meta name="Author" content="LeMyesier">
   <title>Golden Figure</title>
</head>
<body text="#000000" bgcolor="#E6E6FF" link="#0000EE" vlink="#551A8B" alink="#FF0000" background="white.jpg">
&nbsp;
<br>&nbsp;
<br>&nbsp;
<br>&nbsp;
<center>
<p><img SRC="Aurfig.jpg" height=348 width=348>
<br><font color="#999999"><font size=-2>.</font></font>
<br>&nbsp;
<br>&nbsp;
<br>&nbsp;
<p><b><font color="#CCFFFF"><font size=+1><a href="goldfig.htm">Go to the
text</a></font></font></b><b><font color="#CCFFFF"><font size=+1><a href="goldfig.htm"></a></font></font></b>
<p><b><font color="#CCFFFF"><font size=+1><a href="../../seven-planets.htm">Contents</a></font></font></b></center>

</body>
</html>
